march_joint_inertia_controller/include/march_joint_inertia_controller/joint_inertia_controller.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020 Project March.
+
+#ifndef MARCH_HARDWARE_JOINT_INERTIA_CONTROLLER_H
+#define MARCH_HARDWARE_JOINT_INERTIA_CONTROLLER_H
+
+#include <controller_interface/controller.h>
+#include <hardware_interface/joint_command_interface.h>
+#include <pluginlib/class_list_macros.h>
+
+namespace joint_inertia_controller_ns {
```

Comment:
It is already a namespace, so it does not have to be included in the name + it makes it longer.
```suggestion
namespace joint_inertia_controller {
```